### PR TITLE
HARMONY-504  Create helm chart

### PR DESCRIPTION
**Pull request by danquack**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
```
Comment (kband) [inline .github/workflows/test.yml:4]: Nit: misleading name

File: templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+1. Get the pods URL by running these commands:
+kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "github-runner.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[*].metadata.name}"
+
+2. Validate is active
+https://github.com/organizations/{{ .Values.config.owner }}/settings/actions
```
Comment (kband) [inline templates/NOTES.txt:5]: 👍

**Review by kband — CHANGES_REQUESTED** [review]
Add *.tgz to .gitignore
Remove github-runner-0.1.0.tgz from commit

File: templates/NOTES.txt (hunk 2 above)
Comment (danquack) [reply templates/NOTES.txt:5]: There may be a programattic way to handle this, but i wasnt gonna spend a whole lot of time on it

Comment (kband) [reply templates/NOTES.txt:5]: I was loving the deep link to the action runner - it always takes ages to click around to get to it otherwise

**Review by kband — APPROVED** [review]

**Pull request merged**